modules/virt-what-you-can-do-with-virt.adoc

```diff
@@ -6,18 +6,31 @@
 [id="virt-what-you-can-do-with-virt_{context}"]
 = What you can do with {VirtProductName}
 
-{VirtProductName} is an add-on to {product-title} that allows you to run and manage virtual machine workloads alongside container workloads.
+ifndef::openshift-origin[]
+{VirtProductName} provides the scalable, enterprise-grade virtualization functionality in Red{nbsp}Hat OpenShift.
+endif::[]
+ifdef::openshift-origin[]
+{VirtProductName} provides the scalable, enterprise-grade virtualization functionality in {product-title}.
+endif::[]
+You can use it to manage virtual machines (VMs) exclusively or alongside container workloads.
+
+ifndef::openshift-origin,openshift-rosa,openshift-dedicated[]
+[NOTE]
+====
+If you have a {ove-first} subscription, you can run unlimited VMs on subscribed hosts, but you cannot run application instances in containers. For more information, see the subscription guide section about link:https://www.redhat.com/en/resources/self-managed-openshift-subscription-guide#section-8[{ove-first} and related products].
+====
+endif::[]
 
 {VirtProductName} adds new objects into your {product-title} cluster by using Kubernetes custom resources to enable virtualization tasks. These tasks include:
 
-* Creating and managing Linux and Windows virtual machines (VMs)
+* Creating and managing Linux and Windows VMs
 * Running pod and VM workloads alongside each other in a cluster
-* Connecting to virtual machines through a variety of consoles and CLI tools
-* Importing and cloning existing virtual machines
-* Managing network interface controllers and storage disks attached to virtual machines
-* Live migrating virtual machines between nodes
+* Connecting to VMs through a variety of consoles and CLI tools
+* Importing and cloning existing VMs
+* Managing network interface controllers and storage disks attached to VMs
+* Live migrating VMs between nodes
 
-An enhanced web console provides a graphical portal to manage these virtualized resources alongside the {product-title} cluster containers and infrastructure.
+You can manage your cluster and virtualization resources by using the virtualization administrator view in the {product-title} web console or the command line.
```

Comment:
Teeny nit for the purpose of clarity

```suggestion
You can manage your cluster and virtualization resources by using the CLI or the virtualization administrator view in the {product-title} web console.
```

Reply:
Good point! I'm keeping the web console mention first in line (since that's what folks are more likely to use, IIUC). I am hoping this edit adds more clarity, though it's not particularly concise:

> You can manage your cluster and virtualization resources by using the *Virtualization* perspective of the {product-title} web console, and by using the {oc-first}.